src/index.ts

```diff
@@ -117,4 +128,70 @@ export class OpusPool {
     async getRewardsHistory(params: Parameters<typeof rewardsHistory>[1]): Promise<Array<RewardsDataPoint>> {
         return rewardsHistory(this, params);
     }
+
+    /**
+     * Generates mint transaction to mint osTokens from chosen Vault.
+     *
+     * @param params - params for request
+     * @param params.shares - Amount of osTokens to mint
+     * @param params.vault - A vault address
+     * @param params.referrer - Address of the referrer. Optional.
+     * @returns `MintTransactionData` for transaction to sign and broadcast
+     */
+    async buildMintTransaction(params: Parameters<typeof mintOsToken>[1]): Promise<StakingTransactionData> {
+        return mintOsToken(this, params);
+    }
+
+    /**
+     * Retrieves maximum amount of osTokens that can be minted by the user
+     *
+     * @param vault_address - A vault address
+     * @returns Amount of osTokens that can be minted
+     */
+    async getMaxMintForVault(vault_address: Hex): Promise<bigint> {
+        return getMaxMint(this, vault_address);
+    }
+
+    /**
+     * Retrieves health factor for the user
+     *
+     * @param mintedAssets - Amount of osTokens minted by the user
+     * @param stakedAssets - Amount of osTokens staked by the user
+     * @returns Health factor for the user
+     */
+    async getHealthFactorForUser(mintedAssets: bigint, stakedAssets: bigint): Promise<OsTokenPositionHealth> {
+        return getHealthFactor(this, mintedAssets, stakedAssets);
+    }
+
+    /**
+     * Retrieves base data for the pool
+     *
+     * @returns Base data for the pool
+     */
+
+    async getPoolBaseData(): Promise<{ ltvPercent: bigint; thresholdPercent: bigint }> {
```

Comment:
Is it possible to keep this method private.
We'd have difficulties explaining it in the docs, also the naming is very generic and the typedoc for return value as well.